vMenu/menus/VehicleOptions.cs

```diff
@@ -459,6 +466,25 @@ private void CreateMenu()
                         }
                     }
                 }
+                else if (item == speedLimiter) // Speed Limiter Toggled
+                {
+                    SpeedLimited = _checked;
+                    if (_checked)
+                    {
+                        if (vehicle != null && vehicle.Exists())
+                        {
+                            VehicleDefaultMaxSpeed = GetVehicleMaxSpeed(vehicle.Handle);
+                            vehicle.MaxSpeed = vehicle.Speed;
+                        }
+                    }
+                    else
+                    {
+                        if (vehicle != null && vehicle.Exists())
+                        {
+                            vehicle.MaxSpeed = VehicleDefaultMaxSpeed;
```

Comment:
I think it'd be best to use something like GetVehicleModelMaxSpeed, or setting the max speed to 300mph since that's the game's max speed for any entity. 120 mph for planes seems very slow for example.

Reply:
I cannot use GetVehicleModelMaxSpeed or anything of that nature as I am overriding the vehicle model's max speed to a certain value. Unless setting the max speed of a vehicle does not override this?

Reply:
Commits:
d134159
d480d8b